spec/features/web_intake/new_single_filer_spec.rb

```diff
@@ -174,6 +174,11 @@
     expect(page).to have_selector("h1", text: "In 2020, did you have Social Security income, retirement income, or retirement contributions?")
     click_on "No"
 
+    # check for gating logic
```

Comment:
I don't know if this makes sense in a feature spec, but we do some other "check our wizard logic" in the feature specs, so i went ahead and did it. Willing to remove.